pkg/storkctl/schedulepolicy.go

```diff
@@ -88,6 +88,12 @@ func newCreateSchedulePolicyCommand(cmdFactory Factory, ioStreams genericcliopti
 			schedulePolicyName := args[0]
 			var policyItem storkv1.SchedulePolicyItem
 			schedulePolicyType = strings.ToLower(schedulePolicyType)
+			dayOfWeek = strings.ToLower(dayOfWeek)
```

Comment:
I have gone ahead and lowercased the days. I have made sure to add the corresponding lowercase format days in the Days map in apis/stork/v1alpha1/schedulepolicy file which is in turn used in schedule.go where the schedule policy is read.